classes/class-uagb-front-assets.php

```diff
@@ -58,8 +58,55 @@ public static function get_instance() {
 	public function __construct() {
 		add_action( 'wp', array( $this, 'set_initial_variables' ), 99 );
 		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_asset_files' ) );
+
+		if ( is_multisite() && ! is_super_admin() && ! current_user_can( 'manage_network' ) ) {
+			add_filter( 'render_block', array( $this, 'render_icons_dynamically' ), 10, 2 );
+		}
+		
 	}
 
+	/**
+	 * Render icons dynamically for multisite.
+	 *
+	 * @param string $block_content the block data.
+	 * @since x.x.x
+	 * @return string
+	 */
+	public function render_icons_dynamically( $block_content ) {
+		if ( is_admin() || empty( $block_content ) ) {
+			return $block_content;
+		}
+		$pattern                = '/<span\s+class="uagb-mutisite__svg"\s+data-path="([^"]*)"(\s+data-viewbox="([^"]*)")?><\/span>/';
+		$replaced_block_content = preg_replace_callback( $pattern, array( $this, 'replace_svg' ), $block_content );
+		
+		// If the regex errored out, don't replace the $block_content.
+		return ( ! $replaced_block_content || empty( $replaced_block_content ) ) ? $block_content : $replaced_block_content;
+		
+	}
+
+	/**
+	 * Replace the svg with the svg path.
+	 *
+	 * @param array $matches the data of path and viewbox.
+	 * @since x.x.x
+	 * @return string|void
+	 */
+	private function replace_svg( $matches ) {
+		if ( empty( $matches ) ) {
```

Comment:
```suggestion
		if ( ! is_array( $matches ) || empty( $matches[1] || empty( $matches[3]) ) ) {
```
@supriyajjadhav  please check if we can try this.
I think further we just need to return svg rest code can be removed.

Reply:
Resolved. Please check